fre/tests/test_fre_cmor_cli.py

```diff
@@ -41,15 +46,27 @@ def test_cli_fre_cmor_run_opt_dne():
     result = runner.invoke(fre.fre, args=["cmor", "run", "optionDNE"])
     assert result.exit_code == 2
 
+def test_setup_test_files(capfd):
+    ''' set-up test: create binary test files from reduced ascii files in root dir ''' 
+
+    ncgen_input = f'{rootdir}/reduced_ascii_files/reduced_ocean_monthly_1x1deg.199301-199712.sos.cdl'
+    ncgen_output = f'{rootdir}/ocean_sos_var_file/reduced_ocean_monthly_1x1deg.199301-199712.sos.nc'
```

Comment:
`{rootdir}/ocean_sos_var_file` needs to be created

Reply:
yes, good eye